XX-eltoo-onchain.md

```diff
@@ -356,9 +365,10 @@ A node:
   - MAY simply monitor the contents of the most-work chain for transactions.
     - Note: on-chain HTLCs should be sufficiently rare that speed need not be
     considered critical.
-  - MAY monitor (valid) broadcast transactions (a.k.a the mempool).
-    - Note: watching for mempool transactions should result in lower latency
-    HTLC redemptions.
+
+Reacting on transactions broadcasted in the local node mempool is unsafe as a
```

Comment:
may be unsafe?

It might be a watchtower doing the reacting, and if you're spending the settlement outputs to CPFP, nothing is "locked up".

Reply:
Sure "maybe unsafe" as there is a lot of factors to weigh in here.

Well if you're CPFPing an update or settlement, I think you're assuming some fee-bumping reserves are locked up in the post-anchor model. If you're fee-bumping reserve is global (iirc what LND does) someone could exhaust your fee-bumping reserves on your low-value channel A to then attack your high-value channel B.

That said, about all the implications on watchtower infrastructure and fee-bumping reserve management, I don't know if you would like that to be present in this document.